docs/tutorial.md

```diff
@@ -285,6 +285,24 @@ create_keras_neuropod(
     `create_keras_neuropod` runs inference with the test data immediately after creating the neuropod. Raises a `ValueError` if the model output does not match the expected output.
 
 
+### Ludwig
+
+Ludwig contains a utility to export model to the Neuropod format.
+If we train a Ludwig model like this:
+
+```
+ludwig train --data_csv <MY_DATA>PATH> --model_definition <MY_MODEL_DEFINITION> --output_directory <LUDWIG_OUTPUT_DIRECTORY>
+```
+
+We can package it as a Neuropod by running:
+
+```
+python -m ludwig.utils.neuropod_utils --ludwig_model_path <LUDWIG_OUTPUT_DIRECTORY>/<MODEL_NAME>/model --neuropod_path <NEUROPOD_OUTPUT_PATH>
```

Comment:
Can you mention here if output (neuropod model) is going to be a ZIP file or dir by default. Is it controlled by a flag here?

Reply:
it uses the default parameter, so it's True by default, but i can add a flag for it if you believe ti would be useful. In general any suggestion is more than welcome on improving this call: https://github.com/uber/ludwig/blob/master/ludwig/utils/neuropod_utils.py#L277